Gemfile

```diff
@@ -41,7 +41,7 @@ gem 'proofer', github: '18F/identity-proofer-gem', ref: 'v2.8.0'
 gem 'rack-attack', '>= 6.2.1'
 gem 'rack-cors', '>= 1.0.5', require: 'rack/cors'
 gem 'rack-headers_filter'
-gem 'rack-timeout'
+gem 'rack-timeout', require: 'rack/timeout/base'
```

Comment:
what if we made this `require: false` and did the require explicitly in the in the initializer?

Reply:
good call, done in fa1f78f67e53a3d3279e2811945f37d4f8f38a2d